new-log-viewer/src/components/Layout.tsx

```diff
@@ -3,10 +3,40 @@ import React, {
     useContext,
     useEffect,
     useRef,
+    useState,
 } from "react";
 
 import * as monaco from "monaco-editor";
 
+import {
+    Button,
+    DialogContent,
+    DialogTitle,
+    IconButton,
+    Input,
+    Modal,
+    ModalDialog,
+    Sheet,
+    ToggleButtonGroup,
+    Typography,
+    useColorScheme,
+} from "@mui/joy";
+import type {Mode} from "@mui/system/cssVars/useCurrentColorScheme";
+
+import {
+    DarkMode,
+    Description,
+    LightMode,
+    NavigateBefore,
+    NavigateNext,
+    Settings,
+    SkipNext,
+    SkipPrevious,
+    TipsAndUpdates,
+} from "@mui/icons-material";
```

Comment:
Shall we import from the separate files (as done below)? Tree-shaking might not always work, and we might end up with a big bundle that has unneeded imports.